### ISSUE TITLE
options for customized excel output

### ISSUE THREAD

**Issue by shannonpileggi** [pcctc/affirm#35]
create an option that allows users to specify desired columns in summary table of affirmation for first sheet of excel report

also compute an error_pct column

**Comment by Meghansaha** [comment]
Not sure if this needs a separate issue (as I think it relates to this) But we should also address the fact that checks with a error percent of 0 does not print in the summary table. Writing here so I don't forget but willing to open a new issue if you think it's needed.

**Comment by shannonpileggi** [comment]
yeah i am in favor of smaller issues / PRs that tackle specific things, so I suggest a new issue.

**Comment by Meghansaha** [comment]
![image](https://github.com/pcctc/affirm/assets/67963743/3c3870bd-7060-4550-8731-9a6589947628)
